src/marqo/s2_inference/s2_inference.py

```diff
@@ -291,6 +292,19 @@ def _load_model(model_name: str, model_properties: dict, device: str = get_defau
 
     return model
 
+def get_available_models():
+    return available_models
+
+def eject_model(model_name:str,device:str):
+    model_cache_key = _create_model_cache_key(model_name, device)
+    if model_cache_key in available_models:
+        del available_models[model_cache_key]
+        if device.startswith("cuda"):
+            torch.cuda.empty_cache()
```

Comment:
Does this empty the entire CUDA cache? We only want to eject the specified model. 

Also, how do specific CUDA devices (e.g.: `cuda:1`, just `cuda`) affect this, if at all?

Reply:
This will release the cached memory for other applications. 
If we don't call torch.cuda.empty.cached(), the memory will still be reserved for torch use. 
Adding this will also make the results consistent with `nvidia-smi`
This function does not take any argument.

Reply:
Makes sense. I've left a note in the review notes about making an API test for this.